holmes/core/tools.py

```
@@ -134,8 +134,14 @@ def invoke(self, params: Dict) -> StructuredToolResult:
             if hasattr(result, "get_stringified_data")
             else str(result)
         )
+        if len(output_str) == 0:
+            preview = "<empty>"
+        elif len(output_str) > 80:
+            preview = f"{(output_str[:80] + "...")!r}"
```

Pylint (3.3.7)

[error] Parsing failed: 'f-string: unmatched '(' (holmes.core.tools, line 140)'

(E0001)